src/app/shared/base-entity/config.ts

```diff
@@ -2,14 +2,14 @@ import { User } from '@shared/user/models/user';
 import { ClassConstructor, plainToInstance } from 'class-transformer';
 import { isFunction, keys } from 'lodash';
 import { BaseEntityPlain } from './models';
-import { EntityMap } from './types';
```

Comment:
Let's remove `src/app/shared/base-entity/types/entity-map.ts` since it became unused